### ISSUE TITLE
Appveyor taking too long for each build

### ISSUE THREAD

**Issue by antiagainst** [KhronosGroup/SPIRV-Tools#959]
We have tests for VS 2013/2015/2017 and Debug/Release. It now takes ~1h20min for each build. Considering the velocity we have on SPIRV-Tools, it might be nice to remove some of the combinations to avoid waiting for too long.

**Comment by dneto0** [comment]
Good point.  I suspect the Debug implementation on VS2017 is enough to cover all interesting Debug cases.  So I suggest removing Debug VS2013 and Debug VS2015.  What do you think?

**Comment by antiagainst** [comment]
That SGTM.
